ThunderCloud/TSCAuthenticationController.m

```diff
@@ -18,7 +18,8 @@ - (instancetype)init
 {
     if (self = [super init]) {
         
-        [TSCDeveloperModeController shared];
+        TSCDeveloperModeController *devModeController = [TSCDeveloperModeController shared];
+        devModeController.baseURL = devModeController.baseURL;
```

Comment:
Ahhh I remember, it's because `TSCAuthenticationController` has to be the controller which initialises `TSCDeveloperModeController` due to (I think) some race conditions. However just calling the method was causing a compiler warning, so I added in this line